### PR TITLE
completed

### PR DESCRIPTION
**Pull request by pushparaj-c247**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,4 @@
 DB_URl='mongodb://127.0.0.1:27017/User'
-FAKE_URl='mongodb://127.0.0.1:27017/Testing'
\ No newline at end of file
+FAKE_URl='mongodb://127.0.0.1:27017/Testing'
+Port=4000
+Key="ABcdefg"
```
Comment (khushi-c247) [inline .env:4]: remove .env file

File: src/Services/userServices.ts [hunk 2]
```diff
@@ -118,21 +112,15 @@ const login = async (req: Request, res: Response) => {
...
-  //     message: "Invalid password format",
-  //   });
-  // }
+
 
   const passwordMatch = await obj.validatePassword(password);
 
   if (!passwordMatch) {
     return res.status(401).json({ message: "invalid password" });
   }
```
Comment (khushi-c247) [inline src/Services/userServices.ts:121]: Check this condition in validations

**Review by khushi-c247 — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (pushparaj-c247) [reply .env:4]: resolved

File: src/Services/userServices.ts (hunk 2 above)
Comment (pushparaj-c247) [reply src/Services/userServices.ts:121]: resolved

**Pull request merged**